Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AuditDataTransformHelper.java

```diff
@@ -191,14 +190,15 @@ public static CodedValueType createCodeValueType(String code, String codeSys, St
         // If specified, set the User Name
         String userName = null;
         if (userInfo != null && userInfo.getPersonName() != null) {
-            if (userInfo.getPersonName().getGivenName() != null && userInfo.getPersonName().getGivenName().length() > 0) {
+            if (userInfo.getPersonName().getGivenName() != null
+                    && userInfo.getPersonName().getGivenName().length() > 0) {
```

Comment:
Will replace length check of Strings with StringUtils.isNotEmpty(String) on lines of codes that I find them in.